codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSHTTPBindingProtocolGenerator.kt

```diff
@@ -115,49 +108,4 @@ abstract class AWSHTTPBindingProtocolGenerator(
         operationMiddleware.appendMiddleware(operation, OperationEndpointResolverMiddleware(ctx))
         operationMiddleware.appendMiddleware(operation, UserAgentMiddleware(ctx.settings))
     }
-
```

Comment:
The functions below are moved to parent class `HTTPBindingProtocolGenerator` in smithy-swift since they are non-AWS specific.